server/client.go

```diff
@@ -1741,6 +1743,7 @@ func (c *client) generateClientInfoJSON(info Info) []byte {
 	info.CID = c.cid
 	info.ClientIP = c.host
 	info.MaxPayload = c.mpay
+	c.hdrsOk = info.Headers
```

Comment:
Ultimately would probably not be the right place since this function is invoked possibly multiple time at runtime and c.hdrsOk is used in the parser without locking. Since it is planned to be immutable, I would set that when the connection is created (before its parser is started).

Reply:
Very true..

Reply:
Will make that change later today.

Reply:
ok fixed this, will wait to push changes til all of our thoughts are coded.